Doc/library/dbm.rst

```diff
@@ -19,6 +19,13 @@ slow-but-simple implementation in module :mod:`dbm.dumb` will be used.  There
 is a `third party interface <https://www.jcea.es/programacion/pybsddb.htm>`_ to
 the Oracle Berkeley DB.
 
+
+.. seealso::
+
+   Module :mod:`shelve`
+      Persistence module that can store keys and values of types other than bytes.
```

Comment:
Quoting the `shelve` intro text:

> A “shelf” is a persistent, dictionary-like object. The difference with “dbm” databases is that the values (not the keys!) in a shelf can be essentially arbitrary Python objects — anything that the [pickle](https://docs.python.org/3/library/pickle.html#module-pickle) module can handle. This includes most class instances, recursive data types, and objects containing lots of shared sub-objects. The keys are ordinary strings.

The original `.. seealso` text read like this:

>   Module :mod:`shelve`
>       Persistence module which stores non-string data.

I'm not sure if this new description is accurate enough.

Reply:
Kept the original description